packages/patternfly-4/react-charts/src/components/ChartDonut/examples/ChartDonut.md

```diff
@@ -3,58 +3,59 @@ title: 'Donut chart'
 section: 'charts'
 ---
 
-## Blue theme donut chart
+## Blue donut chart
 
-import { ChartDonut, ChartLabel, ChartLegend, ChartTheme } from '@patternfly/react-charts';
-import { Grid, GridItem, Text, TextVariants } from '@patternfly/react-core';
+import { ChartDonut, ChartLabel, ChartLegend, ChartThemeColor, ChartThemeVariant } from '@patternfly/react-charts';
 import './chart-donut.scss';
 
 ```js
 import React from 'react';
+import { ChartDonut, ChartLegend } from '@patternfly/react-charts';
 
-<div style={{ display: 'inline-flex' }}>
-  <div style={{ height: 450, width: 450 }}>
-    <svg style={{ position: 'absolute' }} height={this.size} width={this.size}>
-      <ChartLabel style={{ fontSize: 20 }} text="100" textAnchor="middle" verticalAnchor="middle" x={225} y={200} />
-      <ChartLabel style={{ fill: '#bbb' }} text="Pets" textAnchor="middle" verticalAnchor="middle" x={255} y={250} />
-    </svg>
-    <ChartDonut
-      data={[{ x: 'Cats', y: 35 }, { x: 'Dogs', y: 55 }, { x: 'Birds', y: 10 }]}
-      labels={datum => `${datum.x}: ${datum.y}`}
-      theme={ChartTheme.light.blue}
+<div>
+  <div className="donut-chart-inline">
+    <div className="donut-chart-container">
+      <ChartDonut
+        data={[{ x: 'Cats', y: 35 }, { x: 'Dogs', y: 55 }, { x: 'Birds', y: 10 }]}
+        labels={datum => `${datum.x}: ${datum.y}`}
+        subTitle="Pets"
+        title="100"
+      />
+    </div>
+    <ChartLegend
+      data={[{ name: 'Cats' }, { name: 'Dogs' }, { name: 'Birds' }]}
+      orientation="vertical"
+      y={105}
     />
   </div>
-  <ChartLegend
-    data={[{ name: 'Cats' }, { name: 'Dogs' }, { name: 'Birds' }]}
-    orientation={'vertical'}
-    theme={ChartTheme.light.blue}
-    y={160}
-  />
 </div>
 ```
 
-## Multi-color theme donut chart
+## Multi-color donut chart
 ```js
 import React from 'react';
+import { ChartDonut, ChartLabel, ChartLegend, ChartThemeColor, ChartThemeVariant } from '@patternfly/react-charts';
 
 <div>
-  <div style={{ height: 450, width: 450 }}>
-    <svg style={{ position: 'absolute' }} height={this.size} width={this.size}>
-      <ChartLabel style={{ fontSize: 20 }} text="100" textAnchor="middle" verticalAnchor="middle" x={225} y={200} />
-      <ChartLabel style={{ fill: '#bbb' }} text="Pets" textAnchor="middle" verticalAnchor="middle" x={255} y={250} />
+  <div className="donut-chart-container">
+    <svg className="donut-chart-label">
```

Comment:
Should we use the `title` and `subTitle` props in both examples? Might be confusing for users to see two very different examples like this side by side. At the very least, there should be some explanation of why this example is different than the first one.

Reply:
Wanted to show the option and let the label bake for a bit?

Reply:
I feel like that would just create more confusion in the long run. We could just refer the user to the Victory docs for doing custom graphs like this example.

Reply:
I will remove the example after we're confident we have a good solution for the label

Reply:
Okay, are you saying that this PR is a WIP and that you will remove this before merging? If so we should probably prefix the PR title with '[WIP]' and put a `Do Not Merge` label on it.

Reply:
No, I'm saying there is more work to be done with the theme, but not in this PR.